### ISSUE TITLE
Fails for perl <= 5.24

### ISSUE THREAD

**Issue by eserte** [philiprbrenan/DataTableText#1]
The test suite fails for perls between 5.16 and 5.24 either like this:
```
Warning: Use of "defined" without parentheses is ambiguous at /home/cpansand/.cpan/build/2021072517/Data-Table-Text-20210725-86JoQj/blib/lib/Data/Table/Text.pm line 104.
Compilation failed in require at test.pl line 9.
BEGIN failed--compilation aborted at test.pl line 9.
test.pl .. 
Dubious, test returned 255 (wstat 65280, 0xff00)
No subtests run 
```
or like this:
```
Experimental "my" subs not enabled at /home/cpansand/.cpan/build/2021072516/Data-Table-Text-20210725-nlEG9p/blib/lib/Data/Table/Text.pm line 1905.
Compilation failed in require at test.pl line 9.
BEGIN failed--compilation aborted at test.pl line 9.
test.pl .. 
Dubious, test returned 255 (wstat 65280, 0xff00)
No subtests run 
```

**Comment by philiprbrenan** [comment]
Believed fixed in v20210728.

**Comment by eserte** [comment]
@philiprbrenan In 20210729 I see similar errors.

**Comment by philiprbrenan** [comment]
Possibly fixed in v20210730 by requiring 5.26.0 and above as was previously the case.
